src/main/java/gov/nasa/pds/tools/validate/BundleManager.java

```diff
@@ -46,10 +48,12 @@
 
 public class BundleManager {
 	private static final Logger LOG = LoggerFactory.getLogger(BundleManager.class);
-    private static final Pattern COLLECTION_LABEL_PATTERN =
-      Pattern.compile("(.*_)*collection(_.*)*\\.xml", Pattern.CASE_INSENSITIVE);
-    private static final Pattern BUNDLE_LABEL_PATTERN =
-      Pattern.compile("(.*_)*bundle(_.*)*\\.xml", Pattern.CASE_INSENSITIVE);
+    private static final Pattern COLLECTION_LABEL_PATTERN = Constants.COLLECTION_LABEL_PATTERN; // Ease the requirement to have an underscore after 'collection'.
```

Comment:
@qchaupds not a big deal, but is there a particular reason we are setting a new status variable with a Constant set elsewhere? It just seems like someone should just use the `Constants` class to get these values and not set them again within this class.

like I said, not a huge deal, but seems unnecessary?